Route/RouteIdHandler.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Sonata\AdminBundle\Route;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class RouteIdHandler
```

Comment:
I'd like to see your implementation, with the hash decoding, it will help understand what the purpose of this is, and a name.